terraform/exoscale/modules/kubernetes-cluster/templates/worker-cloud-init.tmpl

```diff
@@ -0,0 +1,26 @@
+#cloud-config
+%{ if ceph_enabled }
+bootcmd:
+- [ cloud-init-per, once, move-second-header, sgdisk, --move-second-header, /dev/vda ]
+- [ cloud-init-per, once, create-ceph-part, parted, --script, /dev/vda, 'mkpart 2 ${ceph_partition_size}GB -1' ]
```

Comment:
`${ceph_partition_size}GB` here is actually how much space to leave before creating the ceph partition, so the variable name becomes misleading. The ceph partition will start at `${ceph_partition_size}GB` and grow until the next partition starts or end of disk :confused:

Reply:
I am happy to take the blame for this confusion. Should be called `rootfs_partition_size` or `non_ceph_partition_size` or `ceph_partition_start`.

Reply:
I'm a bit split to how we should approach this for the upstream version. Should we include this as it is right now *(with the variable name change)* or remove it from the upstreamed version and only have it for the "elastisys" version? Since it kind of forces people

Reply:
I would argue that the `ceph_enabled` flag (I assume it's disabled by default) makes it un-opinionated. I would push it upstream.